src/LuaColor.cpp

```diff
@@ -0,0 +1,213 @@
+// Copyright © 2008-2019 Pioneer Developers. See AUTHORS.txt for details
+// Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+#include "LuaColor.h"
+#include "LuaUtils.h"
+#include "libs.h"
+
+inline void sanity_check(float r, float g, float b, float a)
+{
+	r = std::max(std::min(r, float(255.0)), float(0.0));
```

Comment:
```suggestion
	r = std::max(std::min(r, 255.0f), 0.0f);
```
To reduce keyword noise, please use the `f` suffix literal instead of casting doubles to floats. While the compiler likely will optimize out the cast, it's easier and simpler to use the suffix.

Reply:
Np